### ISSUE TITLE
insertAll tries to insert an additional empty row

### ISSUE THREAD

**Issue by shin-** [shin-/sqlite-wrapper#1]
See https://github.com/dotcloud/stack.io/issues/14

 Here's an example insert statement generated with insertAll:

```
 INSERT INTO permissions (service,method,group_id) SELECT ? AS service, ? AS method, ? AS group_id UNION SELECT ?,?,?;
```

With parameters: `[ 'auth', '.+', 2 ]`

**Comment by ysimonson** [comment]
Any progress with this? It's blocking some commits I want to push to stack.io.

**Comment by shin-** [comment]
Sorry, didn't get much time to code lately! Does [this](https://github.com/shin-/sqlite-wrapper/commit/63e9690702bdb4909de2551c43f09cd303d89f55) fix it? (`npm install sqlite-wrapper@0.2.1`)

**Comment by ysimonson** [comment]
Still getting errors, but I'm pretty sure it's a different one. I think this is fixed; will re-open if not.
